Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,588 @@
+                                                   'modelType': 'url_phishing',
+                                                   'modelExtraInfo': {
+                                                       OOB_MAJOR_VERSION_INFO_KEY: MAJOR_VERSION,
+                                                       OOB_MINOR_VERSION_INFO_KEY: MINOR_DEFAULT_VERSION}})
+    if is_error(res):
+        return_error(get_error(res))
+    return MSG_UPDATE_MODEL % (MAJOR_VERSION, MINOR_DEFAULT_VERSION)
+
+
+def oob_model_exists_and_updated() -> Tuple[bool, int, int]:
+    """
+    Check is the model exist and is updated in demisto
+    :return: book
+    """
+    res_model = demisto.executeCommand("getMLModel", {"modelName": URL_PHISHING_MODEL_NAME})[0]
+    if is_error(res_model):
+        return False, -1, -1
+    existing_model_version_major = res_model['Contents']['model']['extra'].get(OOB_MAJOR_VERSION_INFO_KEY, -1)
+    existing_model_version_minor = res_model['Contents']['model']['extra'].get(OOB_MINOR_VERSION_INFO_KEY, -1)
+    return True, existing_model_version_major, existing_model_version_minor
+
+
+def image_from_base64_to_bytes(base64_message: str):
+    """
+    Transform image from base64 string into bytes
+    :param base64_message:
+    :return:
+    """
+    base64_bytes = base64_message.encode('utf-8')
+    message_bytes = base64.b64decode(base64_bytes)
+    return message_bytes
+
+
+def extract_domainv2(url):
+    ext = no_fetch_extract(url)
+    return ext.domain + "." + ext.suffix
+
+
+def in_white_list(model, url: str) -> bool:
+    """
+    Check if url belongs to the Model whitelist
+    :param model: model which contains top_domains attribute
+    :param url: url to check
+    :return:
+    """
+    if (extract_domainv2(url) in model.top_domains):
+        return True
+    else:
+        return False
+
+
+def get_colored_pred_json(pred_json: Dict) -> Dict:
+    """
+    Create copy and color json values according to their values.
+    :param pred_json: json to color
+    :return: json
+    """
+    pred_json_colored = copy.deepcopy(pred_json)
+    pred_json_colored[MODEL_KEY_SEO] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[MODEL_KEY_SEO][pred_json[MODEL_KEY_SEO]]
+    pred_json_colored[MODEL_KEY_LOGO_FOUND] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[MODEL_KEY_LOGO_FOUND][
+        pred_json[MODEL_KEY_LOGO_FOUND]]
+    pred_json_colored[MODEL_KEY_LOGIN_FORM] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[MODEL_KEY_LOGIN_FORM][
+        pred_json[MODEL_KEY_LOGIN_FORM]]
+    pred_json_colored[DOMAIN_AGE_KEY] = MAPPING_VERDICT_TO_DISPLAY_VERDICT[DOMAIN_AGE_KEY][pred_json[DOMAIN_AGE_KEY]]
+    return pred_json_colored
+
+
+def create_X_pred(output_rasterize: Dict, url: str) -> pd.DataFrame:
+    """
+    Create dataframe to predict from the rasterize output
+    :param output_rasterize: Dict from the output of rasterize command
+    :param url: url to examine
+    :return: pd.DataFrame
+    """
+    website64 = output_rasterize.get('image_b64', None)
```

Comment:
not sure anythong good can be done here, but just to take into account that sometimes the rasterized url is request to collect personal data 
![image](https://user-images.githubusercontent.com/57318201/135427203-61ffdd19-dd54-4194-919a-fcff15c2013e.png)

Reply:
unfortunately nothing ...